tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "allowJs": true,
     "lib": ["es2017", "dom", "webworker"],
     "noImplicitUseStrict": true,
-    "outDir": "local_compiled_js",
+    "outDir": "local_compiled_js_for_test",
```

Comment:
Please remove line 10 & 11, it is not required for typescript checks, right?

Reply:
tsconfig is also used for FE tests that's why line 10 & 11 are needed. (I thinl it doesn't slow down the checks that much and it is always better to have just one source of truth)

Reply:
Ok, sounds good!